<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_profile</class>
 <widget class="QWidget" name="page_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人主页</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelAvatar">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>
       background:#dddddd;
       border-radius:40px;
      </string>
     </property>
     <property name="text">
      <string>头像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLogout">
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelName">
     <property name="styleSheet">
      <string>
       font-size:18px;
       font-weight:bold;
      </string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFollow">
       <property name="text">
        <string>关注 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFans">
       <property name="text">
        <string>粉丝 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelArticles">
       <property name="text">
        <string>文章 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_actions">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnPublish">
       <property name="text">
        <string>✍ 发表文章</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFavorites">
       <property name="text">
        <string>⭐ 查看收藏</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listArticles"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
